api/group/controller/create.js

```diff
@@ -0,0 +1,19 @@
+import Joi from 'joi';
+import saveGroup from '../service/create.js';
+
+const groupSchema=Joi.object({
+  name: Joi.string().required(),
+  permissions:Joi.string().valid('READ','WRITE','DELETE','UPLOAD_FILES','SHARE').required()
```

Comment:
Move permission list to constants
steps:

1. create file api/group/common/constants.js 
2. export permissions array from constants.js 
```
const permissions = [ 'READ','WRITE','DELETE','UPLOAD_FILES','SHARE' ]
export default { permissions }
```
3. import permissions array from constants to api/group/controller/create.js
4. change validation of permissions field to 
`permissions: Joi.string().valid(...permissions).required()`